### DISCUSSION TITLE
Which country is home to prestige watchmaker Rolex?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#123] [Galaxy Trivia] [answered]
**Hint:** 🇨🇭

**Comment by Tynab** [comment, marked as answer]
Rolex is a Swiss luxury watchmaker, and its headquarters and manufacturing facilities are located in the city of Geneva, Switzerland. The company was founded in London, England in 1905, but it moved to Switzerland in 1919, where it has been based ever since. Today, Rolex is one of the most well-known and respected watch brands in the world, famous for its high-quality and innovative timepieces. The company produces around 800,000 watches each year and is known for its strict quality control and attention to detail.
